### DISCUSSION TITLE
Capacitor app: out-of-memory for WebView which lets rendering fail

### DISCUSSION THREAD

**Discussion by jase88** [ionic-team/capacitor#4027] [Ideas]
I recently came across Android Termination Handling API for WebViews
https://developer.android.com/guide/webapps/managing-webview#termination-handle

I think it would be cool if Capacitor could handle such terminations caused by running out-of-memory and log this event. Ideally Capacitor could try to create a new WebView instance at the last known state

**Comment by imhoffd** [comment]
Certainly interesting and worth looking into! Have you run into this in one of your apps?

**Reply by jase88** [reply]
Not yet, but app is used by some low memory Android phones. 

I will have a look how easy it is to simulate a oom error with the Android emulator.
